tests/vmi_multus_test.go

```diff
@@ -665,6 +666,28 @@ var _ = Describe("SRIOV", func() {
 			return
 		}
 
+		createSriovVMI := func(networksNames []string) (vmi *v1.VirtualMachineInstance) {
+			SriovVMI := getSriovVmi(networksNames)
```

Comment:
Is there a reason `SriovVmi` starts with capital letter?

Reply:
No reason.
Fixed.